meinberlin/apps/newsletters/views.py

```diff
@@ -107,10 +105,6 @@ def form_valid(self, form):
                 enabled=True
             ).values_list('creator', flat=True).distinct()
 
-        elif receivers == models.PLATFORM:
```

Comment:
I thought about this, too, but I'm not sure if we really want to delete it. Platform mails are not newsletters, so maybe we still want a way to write a newsletter to all people?

Reply:
Oh never mind, didn't see it's just unused code!